snakedeploy/snakemake_wrappers.py

```
@@ -1,44 +1,139 @@
+        sp.run(
+            ["git", "rev-list", "-1", "HEAD", "--", str(path)],
+            stdout=sp.PIPE,
+            cwd=repo,
+            check=True,
+        )
+        .stdout.decode()
+        .strip()
+    )
+    # get the first git tag that includes this commit:
+    tags = (
+        sp.run(
```

Ruff (0.13.3)

`subprocess` call: check for execution of untrusted input

(S603)